src/main/java/duke/task/EventObj.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * @author leslieharland
+ */
+
+package duke.task;
+
+/**
+ * The Class EventObj.
```

Comment:
It would be better if the java comment could explain what the class EventObj does.